croud/regions/commands.py

```diff
@@ -110,6 +110,25 @@ def regions_delete(args: Namespace) -> None:
     client = Client.from_args(args)
     region_name = args.name
 
+    # Check edge region status
+    regions, errors = client.get("/api/v2/regions/")
+    region = [r for r in (regions or []) if r["name"] in region_name]
```

Comment:
Could maybe do something like this:
```suggestion
    region = next((r for r in (regions or []) if r["name"] == region_name), None)
```
? 
Would immediately get the region or None, and could in fact immediately short-circuit and do a
```
if not region:
    raise ...
```